examples/hybrid_jobs/6_QNSPSA_optimizer_with_embedded_simulator/qnspsa_with_embedded_simulator.ipynb

```diff
@@ -0,0 +1,1360 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    image_uri = retrieve_image(Framework.PL_PYTORCH, region_name)
</pre><p>I realized BASE actually has pennylane. It doesn't look like you use pyTorch so you might consider switching to BASE.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
Good call. I switched to torch container due to the sev2 ticket earlier. Will switch back.